Dockerfile

```diff
@@ -26,10 +28,11 @@ RUN cd / && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o testground
 #::: RUNTIME CONTAINER
 #:::
 
-FROM busybox:1.31.0-glibc
+#FROM busybox:1.31.0-glibc
```

Comment:
??

Reply:
We needed `iptables` installed on the image, and this was the quickest way to get it. Not sure how you install it on `busybox` at the top of my head. Will fix this so that we have a small resulting image.

Reply:
That's fine. Adding iptables to busybox usually involves (a) installing it in debian then (b) copying it and hoping you have the right libc (that's what go-ipfs does).